extensions/amp-beopinion/amp-beopinion.md

```diff
@@ -59,42 +59,46 @@ The `amp-beopinion` component allows you to embed [BeOpinion](https://beop.io/)
 
 Visit the [Placeholders & fallbacks](https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/placeholders) guide to learn more about how placeholders and fallbacks interact for the `amp-beopinion` component.
 
 ## Attributes
 
-<table>
-  <tr>
-    <td width="40%"><strong>data-account (required)</strong></td>
-    <td>The ID of the BeOpinion account (page owner).</td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>data-content (optional)</strong></td>
-    <td>The ID of the BeOpinion content to be displayed on the page.</td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>data-name (optional)</strong></td>
-    <td>The name of the BeOpinion slot on the page.</td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>data-my-content (optional - <code>amp-ad</code> only !)</strong></td>
-    <td>For <code>amp-ad</code> elements of type <code>beopinion</code>, the value can be set to <code>"0"</code> (default value).
-Warning: the <code>amp-beopinion</code> element overrides this value to <code>"1"</code>, to prevent the serving of ads outside of an <code>amp-ad</code> element.</td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>common attributes</strong></td>
-    <td>This element includes <a href="https://amp.dev/documentation/guides-and-tutorials/learn/common_attributes">common attributes</a> extended to AMP components.</td>
-  </tr>
-</table>
+### `data-account`
+
+The ID of the BeOpinion account (page owner).
+
+### `data-content` (optional)
+
+The ID of the BeOpinion content to be displayed on the page.
+
+### `data-name` (optional)
+
+The name of the BeOpinion slot on the page.
+
+[filter formats="ads"]
+
+### `data-my-content` (optional)
+
+For `amp-ad` elements of type `beopinion`, the value can be set to `"0"`
+(default value).
+
+[tip type="important"]
+
+The `amp-beopinion` element overrides this value to `"1"`, to prevent the
+serving of ads outside of an `amp-ad` element.
+
+[/tip]
+
+[/filter]<!-- formats="ads" -->
+
+## Styling
+
+BeOpinion does not currently provide an API that yields fixed aspect ratio for embedded contents. Currently, AMP automatically proportionally scales the content to fit the provided size, but this may yield less than ideal appearance. You might need to manually tweak the provided width and height. Also, you can use the `media` attribute to select the aspect ratio based on the screen width.
 
 ## Validation
 
-See [amp-beopinion rules](https://github.com/ampproject/amphtml/blob/master/extensions/amp-beopinion/validator-amp-beopinion.protoascii) in the AMP validator specification.
+See [amp-beopinion rules](validator-amp-beopinion.protoascii) in the AMP validator specification.
```

Comment:
The relative link works OK?

Reply:
yup yup, they do here :) - amp.dev will turn them into full links